oak_runtime/src/lib.rs

```diff
@@ -411,27 +411,18 @@ impl Runtime {
         let mut node_infos = self.node_infos.write().unwrap();
         let node_info = node_infos.get_mut(&node_id).expect("Invalid node_id");
 
-        let event_details = HandleDestroyed {
-            node_id: node_id.0,
-            handle,
-            channel_id: node_info
-                .abi_handles
-                .get(&handle)
-                .ok_or(OakStatus::ErrBadHandle)?
-                .get_channel_id(),
-        };
-
-        let result = node_info
-            .abi_handles
-            .remove(&handle)
-            .ok_or(OakStatus::ErrBadHandle)
-            .map(|_half| ());
```

Comment:
Imho using an explicit match is also more readable than combining `ok_or` & `map` to convert the option returned by `remove` to the desired `Result`